src/commands/flow/models.ts

```diff
@@ -7,4 +7,12 @@ import {ReleaseNumber} from './ReleaseNumber';
 export interface IBranchDetails extends IGitBranchDetails {
     customer: string;
     version: ReleaseNumber;
-}
\ No newline at end of file
+}
+
+export const PROJECT_PLANNING_PARAM = '-- cf.cplace.projektplanung cf.cplace.rplanRxfImport cf.cplace.integration.ganttAndRiskmanagementAndTaskBoard' +
```

Comment:
This constant should not be in the general `models.ts` file but inside the corresponding command

Reply:
👍